Sources/Filters/General/PaintFilter/PaintFilter.worker.js

```diff
@@ -41,35 +41,37 @@ function handlePaintRectangle({ point1, point2 }) {
 
 function handlePaintEllipse({ center, scale3 }) {
   const radius3 = [...scale3];
+  const indexCenter = center.map((val) => Math.round(val));
```

Comment:
To me it is a `floor` and not a `round`.

 Say `center = [0, 0, 10.6]`, don't you want your slice z=10 to have the full radius instead of slice z=11?

But I guess it depends what slice is currently shown when `z = 10.6`, 10 or 11

Reply:
> don't you want your slice z=10 to have the full radius instead of slice z=11?

well actually, that's what I was thinking we would want. Should we not always paint to the nearest slice ?